packages/paste-website/src/components/search/SearchInput.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import {connectSearchBox} from 'react-instantsearch-dom';
+import {FormInput} from '@twilio-paste/form';
+
+interface SearchFormProps {
+  currentRefinement: string;
+  refine: (value: string) => void;
+  onFocus?: (event: React.FocusEvent<HTMLInputElement>) => void;
+}
+
+const SearchForm: React.FC<SearchFormProps> = ({currentRefinement, refine, onFocus, ...props}) => (
+  <form noValidate action="" role="search" autoComplete="off">
```

Comment:
Just need to add an `onSubmit` event handler on this form to prevent the form submitting and reloading the page when I hit enter in the input